qubes.lua.in

```diff
@@ -170,60 +161,61 @@ function qubes.get_colour_focus(c)
     return get_colour_by_state(c, 'qubes_label_color_focus')
 end
 
-function qubes.manager_menuitem()
-    return {'Qubes Manager', 'qubes-manager',
-        '/usr/share/icons/hicolor/16x16/apps/qubes-logo-icon.png'}
+function is_accessible(path)
+    return os.rename(path, path) and true or false
```

Comment:
Trying to rename the file just to check its existence looks very wrong.

Reply:
I agree. However it seems that standard lua doesn't support a straightforward function to get the job done without additional dependencies.
A few alternatives: https://stackoverflow.com/questions/1340230/check-if-directory-exists-in-lua

Attempting to read or call python/bash/whatever might be another alternative.
I picked rename as it should be fast & even atomic most of the time; it also provides a free permission check.
Just let me know which one you prefer.

Reply:
@woju any idea?